sections/header.liquid

```diff
@@ -343,9 +343,10 @@
     {%- endif -%}
         <a href="{{ routes.root_url }}" class="header__heading-link link link--text focus-inset">
           {%- if section.settings.logo != blank -%}
-            {%- assign image_size = section.settings.logo_width | append: 'x' -%}
-            <img srcset="{{ section.settings.logo | img_url: image_size }} 1x, {{ section.settings.logo | img_url: image_size, scale: 2 }} 2x"
-              src="{{ section.settings.logo | img_url: image_size }}"
+            {%- assign image_size = section.settings.logo_width -%}
```

Comment:
same here. Curious to see what others think too but I feel like we could use `section.settings.logo_width` without the assign but keep the `image_size_2x` one.